website/docs/reference/global-configs/record-timing-info.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Record timing info"
```

Comment:
This one doesn't appear to be on the sidebar. Is that by design?

Reply:
Wow, eagle eyes! 🦅 

Added to the sidebar in https://github.com/dbt-labs/docs.getdbt.com/pull/4814/commits/b29c9d58c6c3129d011cd2659471a80c5c1c168c